index.html

```diff
@@ -136,6 +136,164 @@ <h2>
+        <li>A <dfn>minimum axis value</dfn>, the smallest in-bounds [=logical
+        axis value=].
+        </li>
+        <li>A <dfn>maximum axis value</dfn>, the largest in-bounds [=logical
+        axis value=].
+        </li>
+      </ul>
+      <p>
+        A [=gamepad=] has an <dfn>axis list</dfn> which is a [=list=]
+        containing all the [=axis=] inputs for the [=gamepad=] in some order
+        determined by the [=user agent=].
+      </p>
+      <p>
+        An input control may be designed to automatically return an [=axis=] to
+        a center position when the user stops interacting with the input
+        control. If so, the [=axis=] has a <dfn>preferred axis state</dfn>. An
+        An [=axis=] with a [=preferred axis state=] may also have a <dfn>center
+        position value</dfn> [=input value=] which is the [=logical axis
+        value=] when the [=axis=] is centered.
+      </p>
+      <p>
+        A [=gamepad=] may have [=button=] inputs. A <dfn>button</dfn> is an
+        input control that can be pressed to activate. A [=gamepad=] has a
+        <dfn>button list</dfn> which is a [=list=] containing all the
+        [=button=] inputs for the [=gamepad=] in some order determined by the
+        [=user agent=].
+      </p>
+      <p>
+        An input control may be designed to automatically return a [=button=]
+        to an unpressed state when the user stops interacting with the input
+        control. If so, the [=button=] has a <dfn>preferred button state</dfn>.
+      </p>
+      <p>
+        A [=button=] may have a digital switch to indicate when the [=button=]
+        is activated. If so, the [=button=] has a <dfn>digital button
+        value</dfn> [=input value=] that is `true` when the [=button=] is
+        activated and `false` otherwise.
+      </p>
+      <p>
+        A [=button=] may have an analog sensor enabling the [=button=] to
+        report the degree to which the [=button=] is activated. If so, the
+        [=button=] has these [=input values=]:
+      </p>
+      <ul>
+        <li>An <dfn>analog button value</dfn>, an integer value that represents
+        the degree of activation.
+        </li>
+        <li>A <dfn>minimum analog value</dfn> representing no activation. The
+        smallest in-bounds [=analog button value=].
+        </li>
+        <li>A <dfn>maximum analog value</dfn> representing full activation. The
+        largest in-bounds [=analog button value=].
+        </li>
+      </ul>
+      <p>
+        A [=button=] may be capable of detecting touch. If so, the [=button=]
+        has a <dfn>button touch value</dfn> [=input value=] that is `true` when
+        the [=button=] is touched and `false` otherwise.
+      </p>
+      <p>
+        A [=gamepad=] may have [=touch surfaces=]. A <dfn>touch surface</dfn>
+        is an input control that provides 2D position data representing points
+        of contact. A [=gamepad=] has a <dfn>touch surface list</dfn> which is
```

Comment:
Is it relevant to mention that the we are using a cartesian system to locate these touch points?
```suggestion
        A [=gamepad=] may have [=touch surfaces=]. A <dfn>touch surface</dfn>
        is an input control that provides 2D position data representing points
        of contact in a cartesian coordinate system, where the origin is located in the
        center of the touch surface. A [=gamepad=] has a <dfn>touch surface list</dfn> which is
```

Reply:
Yes, let's note it's a Cartesian system since we only include normalization steps for Cartesian coordinates.

I don't think we can say anything about the [=touch surface=] origin, that's determined by the surface's logical representation which can have arbitrary bounds. The origin is moved to the center by normalization.

Reply:
Makes sense